cmd/helm/list.go

```diff
@@ -153,12 +155,14 @@ func newReleaseListWriter(releases []*release.Release, timeFormat string) *relea
 	elements := make([]releaseElement, 0, len(releases))
 	for _, r := range releases {
 		element := releaseElement{
-			Name:       r.Name,
-			Namespace:  r.Namespace,
-			Revision:   strconv.Itoa(r.Version),
-			Status:     r.Info.Status.String(),
-			Chart:      formatChartname(r.Chart),
-			AppVersion: formatAppVersion(r.Chart),
```

Comment:
why not use formatAppVersion?

Reply:
what i want to know is the chart version instead of appVersion. but currenlty it is not convenient to know. 
for example, for helm list command output are below
#helm list -oyaml
- app_version: 1.16.0
  chart: abc-hello-0.1.0+rc-1.2.0-19
  name: test1
  namespace: default
  revision: "1"
  status: deployed
  updated: 2022-06-15 02:44:41.606385155 -0700 PDT

with output above, it is hard to know what chart versions. i have write speical logic to parse from chart name